src/test/components/ButtonWithPanel.test.js

```diff
@@ -70,6 +72,15 @@ describe('shared/ButtonWithPanel', function() {
       />
     );
```

Comment:
This is a bit optional and more for test styles. Can you move this `render` call to a `setup` function? This could be like this:
```
function setup() {
  return render(
    bla bla bla
  );
}
```

Then we can reuse it elsewhere, for example... for the next request ;)